demos/tests/args.py

```diff
@@ -82,6 +82,24 @@ def resolve(self, context):
         return str(Path(pattern).parent)
 
 
+class DataDirectoryOrigFileNamesArg:
+    def __init__(self, sequence_name):
+        self.sequence_name = sequence_name
+
+    def resolve(self, context):
+        seq_dir = context.data_sequence_dir / self.sequence_name
+        seq = [Path(data.resolve(context))
+            for data in context.data_sequences[self.sequence_name]]
+
+        if not seq_dir.is_dir():
+            seq_dir.mkdir(parents=True)
+
+            for index, data in enumerate(context.data_sequences[self.sequence_name]):
```

Comment:
`data` is unused, and you only use `index` to index `seq`, so you might as well iterate over `seq` instead.